lib/charms/opensearch/v0/opensearch_relation_peer_cluster.py

```diff
@@ -173,6 +174,14 @@ def _on_peer_cluster_relation_changed(self, event: RelationChangedEvent):
             trigger_rel_id=event.relation.id,
         )
 
+        if (
+            deployment_desc.typ == DeploymentType.MAIN_ORCHESTRATOR
```

Comment:
Future: for ticket [DPE-5460](https://warthogs.atlassian.net/browse/DPE-5460), we may want to just resolve this condition at `_on_start`.

Reply:
PS.: this is for awareness only, I copied the link of this comment to the mentioned ticket.